apps/condo/domains/banking/schema/BankIntegrationAccessRight.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Generated by `createschema banking.BankIntegrationAccessRight 'integration:Relationship:BankIntegration:PROTECT; user:Relationship:User:CASCADE;'`
+ */
+
+const { Relationship } = require('@keystonejs/fields')
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/banking/access/BankIntegrationAccessRight')
+const { SERVICE_USER_FIELD } = require('@condo/domains/miniapp/schema/fields/accessRight')
+
+
+const BankIntegrationAccessRight = new GQLListSchema('BankIntegrationAccessRight', {
+    schemaDoc: 'Link between bank integrations and users. The existence of the object means that there is user has access to integration',
+    fields: {
+
+        integration: {
```

Comment:
You should use BankIntegrationContext
```suggestion
        integrationContext: {
```